packages/terra-application-header-layout/package.json

```diff
@@ -46,6 +46,6 @@
     "lint:scss": "stylelint src/**/*.scss",
     "test": "npm run test:jest && npm run test:wdio",
     "test:jest": "jest ./tests/jest/* --config ../../jestconfig.json",
-    "test:wdio": "wdio ../../wdio.conf.js"
+    "test:wdio": "../../node_modules/.bin/wdio ../../node_modules/terra-dev-site/config/wdio/wdio.conf.js"
```

Comment:
We should make this change in the generator

Reply:
Agreed, but the downside is that it's different between core and framework because core uses it's own wdio config and framework uses the default.

Reply:
😞 yeah...